configs/src/deployments/json/linea.json

```
@@ -185,5 +185,177 @@
+    },
+    "GovernanceERC20": {
+      "address": "0x1B6632D1357D26055D6452E3b3BEBd1c1d982e86",
+      "blockNumber": 17120400,
+      "deploymentTx": "0xcdb113f44e53a8b448d29a38b9777d894413cf94e16a47cb2f79745a32ef3363"
+    },
+    "GovernanceWrappedERC20": {
+      "address": "0x6095b5004c59301f8Bb98768Bd395d0bc637C893",
+      "blockNumber": 17120403,
+      "deploymentTx": "0x1969ac1a80310ebcfeb99f53a8bf9a980b21f01d523b1a0832b7943df152632f"
+    },
+    "TokenVotingSetup": {
+      "address": "0x3E773BCb6a8C5ED2f23BF167284DF45D9A550fF6",
+      "blockNumber": 17120406,
+      "deploymentTx": "0x84ee2e0cdfa58d9edfe6c935cc827bf10265168d49a08394f45601bb603c7004"
+    },
+    "TokenVotingSetupImplementation": {
+      "address": "0x9d6db4c3DAfB0886A6B69F821EB69994a81Ab243",
+      "blockNumber": 17120406,
+      "deploymentTx": "0x84ee2e0cdfa58d9edfe6c935cc827bf10265168d49a08394f45601bb603c7004"
+    },
+    "AdminSetup": {
+      "address": "0xeB98a71d69a1e12B62c10368D9dA5364CE0f7178",
+      "blockNumber": 17119360,
+      "deploymentTx": "0x7f73a29f6535f90fee801cc097b71cce03b469b2f5a57d4d9edb091f5df84091"
+    },
+    "AdminSetupImplementation": {
+      "address": "0xfD959A00ff63750Fd295A801eBF3C9FA2f22bF81",
+      "blockNumber": 17119360,
+      "deploymentTx": "0x7f73a29f6535f90fee801cc097b71cce03b469b2f5a57d4d9edb091f5df84091"
+    },
+    "MultisigSetup": {
+      "address": "0xE47C0B1Ea698F2914b6CC218cdC3A5Ca4FAcff18",
+      "blockNumber": 17119966,
+      "deploymentTx": "0xb8cf83af3cb4ee8c691fd9a9df1725ea20faf5250473190cfb5ba4c81eefa5ba"
+    },
+    "MultisigSetupImplementation": {
+      "address": "0x10a9F51c7e163ac2d061eE8EcD72557722126A81",
+      "blockNumber": 17119966,
+      "deploymentTx": "0xb8cf83af3cb4ee8c691fd9a9df1725ea20faf5250473190cfb5ba4c81eefa5ba"
+    },
+    "PlaceholderSetup": {
+      "address": "0xC9d0AaA13DECA0486016E3DD9aEe02bf2FdD76dC",
+      "blockNumber": 12678810,
+      "deploymentTx": "0x7d26f9386f9df360c975f8f503affdb56ea5c85a59125390c38f3145f45f9df0"
+    },
+    "TokenVotingRepoProxy": {
+      "address": "0x9C08F499083963F69c1D51D5e3031afC111CD8f7",
+      "blockNumber": 12678824,
+      "deploymentTx": "0x992289d7c48fa1743f59169f3bc90d4b24dab7647fa3a90261d712de07ebfc0f"
+    },
+    "AdminRepoProxy": {
+      "address": "0x7d56667664ADf9e5aAb436629e6a789162Ad83eA",
+      "blockNumber": 12678833,
+      "deploymentTx": "0x37a2a89ce934b8d3b8e5d8ecd0ff597343795fe69828ff7cc792b1e3a4cf51ca"
+    },
+    "MultisigRepoProxy": {
+      "address": "0x2667636E0f5eA63c771509e1d3377177E9Da056D",
+      "blockNumber": 12678840,
+      "deploymentTx": "0xa2ac597c0c9e7534c85556d067bb96c47f68323951e20d043da06f2283f0ed52"
+    },
+    "StagedProposalProcessorRepoProxy": {
+      "address": "0x1f965E73Aa062a161d27f8B0274C14cb06DC70D7",
+      "blockNumber": 17121280,
+      "deploymentTx": "0xf2d594F3C93C19D7B1a6F15B5489FFcE4B01f7dA"
```

This is the management DAO proxy address
```suggestion
      "deploymentTx": "0x17f04670d3c16db4c01800809d1a6535b65abadfee9b7476e37c485af7732bd1"
```